packages/hash/api/src/db/adapter.ts

```diff
@@ -248,116 +247,6 @@ export interface DbAdapter extends DataSource, DbClient {
  * Generic interface to the database.
  */
 export interface DbClient {
```

Comment:
Are we leaving this in for now because removing it outright would require changing too much in one PR?

Reply:
Yes, we'd have TS errors everywhere in other model classes

Reply:
Makes sense